ui/component/core/src/index.ts

```diff
@@ -109,6 +109,37 @@ export enum MapType {
     RASTER = "RASTER"
 }
 
+export type BaseMapMarkerConfig = {
```

Comment:
Doesn't seem like my previous comment has been implemented. Don't want this config in the core component it needs to be part of the `ManagerAppConfig`